caffeine/main.py

```diff
@@ -276,6 +282,10 @@ def __init__(self, show_preferences=False):
 
         builder.connect_signals(self)
 
+        if activate_on_startup:
```

Comment:
Would like you to review how it checks for the need of activation. Apart from that it is ready to merge.